src/licensedcode/plugin_licenses_reference.py

```diff
@@ -39,7 +43,10 @@ class LicensesReference(PostScanPlugin):
     """
     Add a reference list of all licenses data and text.
     """
-    codebase_attributes = dict(licenses_reference=attr.ib(default=attr.Factory(list)))
+    codebase_attributes = dict(
+        license_references=attr.ib(default=attr.Factory(list)),
+        licensedb_references=attr.ib(default=attr.Factory(list))
```

Comment:
I would like to discuss this licensedb_references further ... I am not sure I get it entirely.

Reply:
Here there are two levels of de-referencing. One is licenses with a unique license-key, the another is a matched rule/license with a unique rule identifier.